package.json

```
@@ -26,7 +26,7 @@
     "tslib": "^1.9.3"
   },
   "devDependencies": {
-    "@angular/compiler": "6.0.0",
+    "@angular/compiler": "8.2.13",
```

Minimum required version is preferred:

```suggestion
    "@angular/compiler": "8.2.0",
```

---

Done.